contents/handbook/people/team-structure/ingestion.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Team Ingestion
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+Team ingestion is part of the [Platform team](/handbook/people/team-structure/platform) at PostHog.
+
+## People
+
+- Tiina Turban (Full Stack Engineer)
+- Yakko Majuri (Full Stack Engineer)
+
+## Mission statement
+
+Provide the best events pipeline in the world.
+
+## Responsibilities
+
+Team Ingestion owns our ingestion pipeline end-to-end. That means we own client libraries, the Django server ingestion API, the ingestion (plugin) server, as well as our Kafka and ClickHouse setup, where it pertains to event ingestion.
```

Comment:
when we say we own client libraries, could we clarify that the ingestion part and not feature flags (or do we own feature flags too?
```suggestion
Team Ingestion owns our ingestion pipeline end-to-end. That means we own the Django server ingestion API, the ingestion (plugin) server, as well as our client libraries, Kafka and ClickHouse setup, where it pertains to event ingestion.
```